hack/run-functests.sh

```diff
@@ -1,11 +1,20 @@
 #!/bin/bash
 
+which go
+if [ $? -ne 0 ]; then
+  echo "No go command available"
+fi
+
+GOPATH="${GOPATH:-~/go}"
+PATH=$PATH:$GOPATH/bin
```

Comment:
I'd export it to avoid nasty surprises in the future:
https://stackoverflow.com/questions/1158091/defining-a-variable-with-or-without-export

Reply:
Thanks!